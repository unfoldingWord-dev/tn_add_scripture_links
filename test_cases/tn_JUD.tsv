Reference	ID	Tags	SupportReference	Quote	Occurrence	Note
1:1	abc1		rc://*/ta/man/translate/translate-names	Jude	1	Compare with 2 Peter 3:3 and see verses 17-18.
1:4	abc2		rc://*/ta/man/translate/figs-metaphor	condemnation	1	This refers to 2 Peter 2:1-3 and verse 6.
1:9	abc3		rc://*/ta/man/translate/translate-unknown	Michael	1	See Daniel 10:13 and Revelation 12:7.
