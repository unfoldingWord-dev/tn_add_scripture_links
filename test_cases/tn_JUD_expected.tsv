Reference	ID	Tags	SupportReference	Quote	Occurrence	Note
1:1	abc1		rc://*/ta/man/translate/translate-names	Jude	1	Compare with [2 Peter 3:3](../../2pe/03/03.md) and see [verses 17-18](../01/17.md).
1:4	abc2		rc://*/ta/man/translate/figs-metaphor	condemnation	1	This refers to [2 Peter 2:1-3](../../2pe/02/01.md) and [verse 6](../01/06.md).
1:9	abc3		rc://*/ta/man/translate/translate-unknown	Michael	1	See [Daniel 10:13](../../dan/10/13.md) and [Revelation 12:7](../../rev/12/07.md).
